docs/running-jobs/interactive-jobs.md

```diff
@@ -43,6 +45,41 @@ Expand the SSH tab on the left side of the application and click X11.
 
 In the X11 Menu check the "Enable X11 Forwarding" checkbox and type "localhost:0" in the X display location field.  Clicking open will open a terminal window where you can login.
 
+##### Command Prompt, Powershell
+
+This tutorial assumes that you are already set up with OpenSSH for Windows. [Here](https://learn.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse?tabs=gui) is a guide for setting up OpenSSH for Windows. 
+
+You can deploy Xming by running the Xlaunch application (or opening the Xlaunch shortcut on your desktop). When prompted to select display settings, select 'Multiple windows' and set the Display number to 0. From there, use the default selections until Xlaunch is finished.
+
+Next, you'll need to set your display value. Open the Command Prompt (or Powershell) and set the $DISPLAY variable as follows: 
+
+Command Prompt: 
+```
+cd ../../
+set DISPLAY=127.0.0.1:0.0
+```
+
+Powershell:
+```
+cd ../../
```

Comment:
This should be an absolute path so that people know where they need to navigate to, even if they aren't in a fresh Command Prompt/Powershell window.